packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt

```diff
@@ -108,3 +108,10 @@
 # all values below this line were inserted from config.txt.bk (your old config)
 # and can be merged with the above values
 ################################################################################
+
+################################################################################
+# Include distribution specific config file if it exists.
+################################################################################
+
+[all]
+include distroconfig.txt
```

Comment:
I can see the logic in that, however if a user moved from upstream LE to a custom distribution then the overrides would require a full clean install. Thats why I pitched having it in the upstream config file. That way we could avoid shipping our own config.txt and only ship a distroconfig.txt file in the case we wanted to override the upstream version.